src/app/zap-templates/partials/header.zapt

```diff
@@ -1,6 +1,6 @@
 /*
  *
- *    Copyright (c) 2021 Project CHIP Authors
+ *    Copyright (c) 2020-2021 Project CHIP Authors
```

Comment:
I really wish this was done in a different PR to not pollute the diff of this one. It has introduced a bunch of unrelated changes :(